fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -2798,6 +2803,422 @@ The possible states of a bulk transfer can be seen in [Figure 62](#figure-62).
+
+If the currency conversion execution request specified that the conversion was dependent on the outcome of a payment by specifying a determining transfer ID (see [Table 53](#table-53)), then the FXP should not return a status COMPLETED, but rather a status RESERVED. This tells the caller (and the optional switch, if there is one) that the FXP has approved the conversion and will await the outcome of the linked payment before finalising its ledgers.
+
+###### Table 54
+
+| **Name** | **Cardinality** | **Type** | **Description** |
+| --- | --- | --- | --- |
+| **fulfilment** | 0..1 | ilpFulfilment | The fulfilment of the condition specified for the currency conversion. Mandatory if the conversion has been executed successfully. |
+| **completedTimeStamp** | 0..1 | DateTime | Time and date when the conversion was finalised. |
+| **conversionState** | 1 | TransferState | The current status of the conversion execution request.  |
+| **extensionList** | 0..1 | ExtensionList | Optional extension list, specific to the deploymeny. |
+
+**Table 54 -- PUT /fxTransfers/_{ID}_ data model**
+
+#### 6.13.4.2 PATCH /fxTransfers/_{ID}_
+
+Alternative URI: N/A
+
+Logical API service: **Update Currency Conversion Execution Information**
+
+The callback **PATCH /fxTransfers/**_{ID}_ is used to inform the client of the final result of a currency conversion execution request. The _{ID}_ in the URI should contain the **commitRequestId** (see [Table 53](#table-53)) that was used for the creation of the currency conversion execution request, or the _{ID}_ that was used in the [**GET /fxTransfers/**_{ID}_](#61331-get-fxtransfers). See [Table 55](#table-55) for data model.
+
+Specifically, it is used by the finalising entity (either the optional switch or the payee FSP) to inform an FXP of the finalised status of a currency conversion where the currency conversion is dependent on the finalised status of a determining transfer. The finalising entity will send a **PATCH /fxTransfers/_{ID}_** message to the FXP to signal the final status of the associated payment, so that the FXP can adjust its ledgers accordingly.
+
+###### Table 55
+
+| **Name** | **Cardinality** | **Type** | **Description** |
+| --- | --- | --- | --- |
+| **completedTimeStamp** | 0..1 | DateTime | Time and date when the conversion was finalised. |
+| **conversionState** | 1 | TransferState | The final status of the conversion execution request.  |
+| **extensionList** | 0..1 | ExtensionList | Optional extension list, specific to the deploymeny. |
+
+**Table 55 -- PUT /fxTransfers/_{ID}_ data model**
+
+#### 6.12.5 Error Callbacks
+
+This section describes the error callbacks that are used by the server
+under the resource **/fxTransfers**.
+
+#### 6.13.5.1 PUT /fxTransfers/_{ID}_/error
+
+Alternative URI: N/A
+
+Logical API service: **Return Currency conversion Execution Information Error**
```

Comment:
"**Return Currency conversion Execution Information Error**" -> "**Return Currency Conversion Execution Information Error**" (capital C in "Conversion")